news/_posts/2022-01-05-multistore-configuration-forms.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Multistore configuration forms"
+subtitle: "How multistore configuration forms got better with PrestaShop 1.7.8"
+date: 2022-01-05 
+authors: [matthieu-rolland]
+icon: icon-puzzle
+tags: [multistore, contribute]
+---
+
+Did you know that PrestaShop has a cool feature called "multistore", which is the ability to manage multiple shops from the same back office ?
+
+One of the back-office aspects of this feature is that many configuration forms can have different values for different shops, which is dealt with using shop contexts.
+
+## How it works and what was less than stellar about it
+
+In a configuration form, a merchant can set different values for different shop contexts, of which there are three, in this order of inheritence:
+
+- All shop context
+- Group shop context
+- SIngle shop context
+
+![Shop contexts](/assets/images/2022/01/shopcontexts.png)
+
+When a value is not set for a context, it inherits the value of the parent context. Consequently, the "All Shops" context values can be considered a default values. 
```

Comment:
Maybe a small example below would help, like a form with 3 inputs (add a screenshot), where merchant overrides 1 value so we say
- value of input 1 is A because it's the value defined for All shops
- value of input 2 is B because it's the value defined for All shops
- value of input 3 is C because it's the value defined for only the current shop